roofit/roofitcore/inc/RooAbsData.h

```diff
@@ -362,7 +362,7 @@ class RooAbsData : public TNamed, public RooPrintable {
   RooArgSet _vars;         ///< Dimensions of this data set
   RooArgSet _cachedVars ;  ///<! External variables cached with this data set
 
-  RooAbsDataStore* _dstore = nullptr; ///< Data storage implementation
+  std::unique_ptr<RooAbsDataStore> _dstore; ///< Data storage implementation
```

Comment:
Do we need an I/O conversion rule here to read file written with the previous version, or it is done automatically for raw pointer -> unique_pointer ?

Reply:
This is done automatically, we have done the same in many classes before and Philippe also told me explicitly that it can be done